### PR TITLE
Add: Drupal chapter

### PR DESCRIPTION
**Pull request by Mexidense**
Hi guys!

I've added Drupal chapter with resources list from #16 

I don't know if I miss any thing... :smile: 

Thank you!

### CODE REVIEW

File: programming/DRUPAL_8.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Drupal 8 good practices, standards and resources
+==
```
Comment (Haru0) [inline programming/DRUPAL_8.md]: Let's unify this. Do you think we could use only a single equal sign as this does no difference?
> I was using three everywhere, but I can revert that anytime.

File: programming/DRUPAL_8.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# Table of contents
+
+* General information
```
Comment (Haru0) [inline programming/DRUPAL_8.md]: I prefer ordered lists somehow. I'm cleaning up everything in the #13.  Do you think unordered lists are better?

File: programming/DRUPAL_8.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* Standards
+* Good practices
+* IDE configuration
+
+# General information
+
```
Comment (Haru0) [inline programming/DRUPAL_8.md]: It would be nice to have some introduction at the beginning of each chapter.

**Comment by Haru0** [comment]
Thanks for this pull request @Mexidense :heart: !

File: programming/DRUPAL_8.md (hunk 2 above)
Comment (Mexidense) [reply programming/DRUPAL_8.md]: I would prefer to use ordered list! :+1:  :ok_hand: Changed it!

**Pull request merged**